src/scss/common.scss

```diff
@@ -1,6 +1,6 @@
 .container {
   width: 100%;
-  max-width: 100rem;
+  /*max-width: 100rem;*/
```

Comment:
I'm not really sure this is a great idea. People with extra wide monitors don't like full width web pages (I think @CodingJWilliams showed a screenshot on Discord as an example). Also removing the max-width here basically makes the `container` class redundant - I'd want to remove it from every page if that's the case.

We can make it so that only the editor gets the full width, but I still don't think it needs it there, either. It's not like it needs it tbh.

Reply:
The issue is the styling then looks odd when the page is cut off (gray borders down either side)

I actually feel the opposite - no harm in the pages filling the window. If people with super huge monitors want to limit the size, then they can just resize their windows?

Reply:
I was going to try and see how to make it look better, with the editor in particular.

> If people with super huge monitors want to limit the size, then they can just resize their windows?

The moment this happens means that the UI has failed :P